x-pack/plugins/security_solution/server/lib/detection_engine/fleet_integrations/api/get_installed_integrations/installed_integration_set.ts

```diff
@@ -134,6 +178,20 @@ const getIntegrationsInfoFromPolicy = (
       is_enabled: input.enabled,
     };
   });
+
+  // Base package may not have policy template, so pull directly from `policy.package` if so
+  return [
+    ...integrationInfos,
+    ...(policy.package
+      ? [
+          {
+            integration_name: policy.package.name,
+            integration_title: policy.package.title,
+            is_enabled: true, // Always true if `policy.package` exists since this corresponds to the base package
+          },
```

Comment:
This was the 🎟️ to actually fixing the base integration showing as `Installed: enabled` if there was an integration policy available. Combined with removing `version` from the `packageKey` we can now display an accurate integration `title`, and `is_enabled` status.

Reply:
> This was the 🎟️ to actually fixing the base integration showing as `Installed: enabled` if there was an integration policy available. Combined with removing `version` from the `packageKey` we can now display an accurate integration `title`, and `is_enabled` status.

That is a great fix, thank you for that @spong!

Reply:
> Though note, we need to get to the bottom of multi-version integration policies, or rather supporting many integration policies of different versions, and what that actually looks like in the UI since these can differ from the actual installed version of the integration itself. I couldn't find much info in the docs around how older versioned policies behave against a new integration itself, so some open questions here still, but things seem better in testing so far at least.

@spong Could you please open a discussion ticket for this?

Reply:
Here we go! https://github.com/elastic/kibana/issues/152407

Also added to this weeks SPMWG meeting as well 🙂